src/pages/dave.cljs

```diff
@@ -38,36 +38,31 @@
 
   (layout.content-block/content-inner
    (h/h2 "Experience")
-
    (h/h3 "Commercial")
-   (layout.body-text/p (h/strong "10+ years total") " professional web experience.")
-   (layout.body-text/p (h/strong "Started out") " making cool MySpace pages for local indie bands.")
-   (layout.body-text/p (h/strong "3 years building") " " (wheel.link.hoplon/external "http://furstmedia.com.au/" "Furst Media's") " publishing platform (10% MoM growth).")
-   (layout.body-text/p (h/strong "5 years owning and operating") " my previous digital agency, " (wheel.link.hoplon/external (wheel.abn.core/abr-search-url "58147061121") "Godel") ".")
-   (layout.body-text/p (h/strong "2 years as chief solutions architect") " at " (wheel.link.hoplon/external "http://equiem.com.au/" "Equiem") " " (wheel.link.hoplon/external "http://equiem.com.au/2016" "(100% YoY growth)") ".")
-   (layout.body-text/p (h/strong "Current owner and CTO") " of Dim Valley and Nice Feature.")
-
+   (h/ul
```

Comment:
these "raw" ul/li tags will not get the benefit of width capping that `layout.body-text/p` provides